e2e/common/userRegistration.js

```diff
@@ -0,0 +1,56 @@
+const getRegistrationUserData = () => ({
+  email: `e2e.user.${Date.now()}@example.com`,
+  password: 'secret',
+  firstName: 'User',
+  lastName: `For testing ${Date.now()}`,
+});
+
+const registerUser = async (userData = {}) => {
+  const { email, password, firstName, lastName } = {
+    ...getRegistrationUserData(),
+    ...(userData || {}),
```

Comment:
In arguments, you set the default value for `userData`. Can we'll remove `|| {}`?

Reply:
Yep